### PR TITLE
Making the Python 3 package pip-installable

### PR DESCRIPTION
**Pull request by emanuelen5**
Makes the repository installable with pip, either by cloning and installing the project, or installing directly from the git-repo:

```bash
pip install git+https://github.com/emanuelen5/XPlaneConnect.git  # or https://github.com/nasa/XPlaneConnect.git if this PR becomes merged.
```

Note that this is only for Python 3.

I'm not sure what version you would like, I set it to `1.0.0`, but perhaps `1.3rc6` makes more sense (if it should follow the same release pattern as the XPC plugin).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name="xplaneconnect",
+    version="1.0.0",
+    python_requires=">=3",
+    packages=find_packages("Python3/src"),
+    package_dir={"": "Python3/src"},
+    description="XPlaneConnect (XPC) facilitates communication to and from the XPCPlugin.",
```
Comment (teubert) [inline setup.py:10]:
```suggestion
    description="XPlaneConnect (XPC) facilitates communication to and from the XPCPlugin. Note: To use the XPC package, you must first install the XPCPlugin. See https://github.com/nasa/XPlaneConnect/wiki/Getting-Started for instructions on installing the XPCPlugin",
```
I'm afraid that users will install this from pip without understanding that they must first install the XPCPlugin. Hopefully this note will help prevent that.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    url="https://github.com/nasa/XPlaneConnect/",
```
Comment (teubert) [inline setup.py:11]:
```suggestion
    url="https://github.com/nasa/XPlaneConnect/",
    project_urls={  # Optional
        'Bug Reports': 'https://github.com/nasa/xplaneconnect/issues',
        'Organization': 'https://www.nasa.gov/content/diagnostics-prognostics',
        'Source': 'https://github.com/nasa/xplaneconnect',
        'Documentation': 'https://github.com/nasa/XPlaneConnect/wiki'
    }
```
Let's add a few more links here

**Review by teubert — APPROVED** [review]
This is a great idea, thanks for your contribution. 

I made a few suggestions to add to the description for PyPI.

**Comment by emanuelen5** [comment]
I think those additions are good. :)

What's your opinion on the version and package name? As I configured it now, the package will be installed with the name "xplaneconnect", but it will be importable as `xpc`, i.e.

```bash
$ pip list
Package       Version
------------- -------
pip           21.3.1
setuptools    58.4.0
wheel         0.37.0
xplaneconnect 1.0.0

$ python
>>> from xpc import XPlaneConnect
```

**Comment by jason-watkins** [comment]
I think the name is good. The version should match the current XPC version, so 1.3rc6.

We will also need a CLA on file before we can merge. If you could, please fill out the appropriate CLA [here](https://github.com/nasa/XPlaneConnect/tree/master/Contributor%20Agreements) and email it to me (jason.watkins@nasa.gov).

**Comment by emanuelen5** [comment]
Awesome. Fixed.

**Comment by jason-watkins** [comment]
Thanks for your contribution!

**Pull request merged**